.github/workflows/upload_release_assets.yml

```diff
@@ -25,3 +25,18 @@ jobs:
           asset_path: ./assets/ipfix-collector.yaml
           asset_name: ipfix-collector.yaml
           asset_content_type: application/octet-stream
+  build-consumer:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build assets
+        run: cp ./build/yamls/kafka-server.yaml ./assets
+      - name: Upload kafka-server.yaml
```

Comment:
let's call kafka-server.yaml as kafka-flow-collector.yaml to be more specific and it could cover both broker and consumer?